src/Main/jkEpisode.c

```diff
@@ -40,6 +40,7 @@ void jkEpisode_Shutdown()
 
 int jkEpisode_LoadVerify()
 {
+    strcpy(jkRes_curDir, "sdmc:/jk");
```

Comment:
InstallHelper should probably also help here as well, it just needs the one chdir